src/MongoDB.Bson/BsonDefaults.cs

```diff
@@ -27,8 +28,9 @@ public static class BsonDefaults
         private static bool __dynamicArraySerializerWasSet;
         private static IBsonSerializer __dynamicArraySerializer;
         private static bool __dynamicDocumentSerializerWasSet;
-        private static IBsonSerializer __dynamicDocumentSerializer; 
+        private static IBsonSerializer __dynamicDocumentSerializer;
         private static GuidRepresentation __guidRepresentation = GuidRepresentation.CSharpLegacy;
+        private static GuidRepresentationMode __guidRepresentationMode = GuidRepresentationMode.V2;
```

Comment:
The new modes are:

V2 = Guids are handled like they have been up to now in V2 versions of the driver
V3 = Guids are handled the new way